specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1399,15 +1660,174 @@
         },
         "isCurrent": {
           "type": "boolean",
-          "description": "Indicates if API revision is current api revision."
+          "description": "Indicates if API revision is current api revision.",
+          "readOnly": true
         },
         "isOnline": {
           "type": "boolean",
-          "description": "Indicates if API revision is accessible via the gateway."
+          "description": "Indicates if API revision is accessible via the gateway.",
+          "readOnly": true
         }
       },
       "description": "API base contract details."
     },
+    "ApiRevisionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiRevisionContract"
+          },
+          "description": "Page values."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "Paged Api Revision list representation."
+    },
+    "ApiRevisionContract": {
+      "properties": {
+        "apiId": {
+          "type":"string",
+          "readOnly": true,
+          "description": "Identifier of the API Revision."
+        },
+        "apiRevision": {
+          "type":"string",
+          "readOnly": true,
+          "description": "Revision number of API."
+        },
+        "createdDateTime": {
+          "type":"string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time the API Revision was created."
```

Comment:
does is follow ISO 8601 guidelines, add comment "The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard."

Reply:
It is just for display.  I can add that text.